src/Features/Core/Portable/Emit/CompilationOutputFilesWithImplicitPdbPath.cs

```diff
@@ -1,15 +1,15 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
+
 using System;
 using System.IO;
 using System.Linq;
 using System.Reflection.PortableExecutable;
 using Microsoft.CodeAnalysis.Debugging;
-using Microsoft.CodeAnalysis.Emit;
 using Roslyn.Utilities;
 
-namespace Microsoft.VisualStudio.LanguageServices.Implementation.ProjectSystem
+namespace Microsoft.CodeAnalysis.Emit
```

Comment:
Why moving this here versus just into EditAndContinue? Does anything else use it at this point? Would prefer we kept it in that namespace until we had a broader need.

Reply:
CompilationOutputs are in CodeAnalysis.Emit. It's Emit artifact, not specific to EnC. Even though it's currently used only by EnC.